api/bertytypes.proto

```diff
@@ -985,3 +985,11 @@ message SystemInfo {
     string system_username = 28;
   }
 }
+
+message Libp2pPeers {
```

Comment:
btw, if we can't have more data than what libp2p can return, yes, name this `libp2p-peers`, but as soon as we know that we will enrich this data with additional metadata, this call should be named `p2p-peers` and the peers should be a struct containing the address, but also additional data